### DISCUSSION TITLE
TLS Africa

### DISCUSSION THREAD

**Discussion by Fadviewguy** [iptv-org/community#710] [Q&A]
**Channel Name:** TLS Africa
**Channel ID:** TLSAfrica.tg
**Country:** Togo
**Language:** French
**Website:** https://tlsafrica.tg/
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by BellezaEmporium** [comment]
https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://dai.ly/x815p43

https://iptv.dc-team.com/query?url=https://dai.ly/x815p43

**Comment by jkemp814** [comment]
second link works.

Streamlink throwing application error.

**Comment by BellezaEmporium** [comment]
Yup my server is getting crazy right now.

**Comment by jkemp814** [comment]
Where do I learn how to set up a server? Step by step.

**Comment by BellezaEmporium** [comment]
Mine is hosted on Heroku.

First you need to create an Heroku account
Then you see which code you wanna run (see https://github.com/LaneSh4d0w/query-streamlink for an example)
Create a Procfile that will indicate Heroku which worker is going to be assigned and how will it work.
Then, create a new program under Heroku, give him the name you want, and precise the location you want to assign to it.

After this, go to the Deploy settings, link your GitHub account to Heroku, choose your project and deploy !

**Comment by jkemp814** [comment]
Excellent, let me start working on it.

**Comment by Fadviewguy** [comment]
The two links  work perfect for me.

**Comment by Alstruit** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x815p43

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19693
